tests/unit/AccountancyHelpersTest.php

```diff
@@ -51,4 +53,70 @@ public function testDateRange(Date $end, string $expectedResult) : void
         ]);
         $this->assertSame($expectedResult, $res);
     }
+
+    /**
+     * @return mixed[]
+     */
+    public function getPrices() : array
+    {
+        return [
+            [1.12, '1,12'],
+            [2.451, '2,45'],
+            [3.459, '3,46'],
+            ['4', '4,00'],
+            ['5a', '5,00'],
+            ['a6', '0,00'],
```

Comment:
Tady bych osobně radši házel výjimku - tohle totiž nikdy nemůže přijít jako platná hodnota a 100% je to bug